docs/documentation/getting-started/deployment/infrastructure-as-code/azure-private.md

```diff
@@ -0,0 +1,192 @@
+# Local Deployment using Infrastructure as Code
+
+This guide describes how to deploy a two-party Carbyne Stack Virtual Cloud
+onto an Azure private Kubernetes cluster using Infrastructure as Code (IaC).
+
+## Setup
+
+1. Clone the carbynestack/carbynestack repository
+
+    ```shell
+    git clone git@github.com:carbynestack/carbynestack.git
+    ```
+
+2. Change into the `carbynestack` directory
+
+    ```shell
+    cd carbynestack
+    ```
+
+3. Checkout the tag `sdk-v0.5.0`
```

Comment:
needs to be updated once [carbynestack/#103 ](https://github.com/carbynestack/carbynestack/pull/103) has been merged and released